cmd/livepeer_cli/wizard.go

```diff
@@ -307,7 +307,7 @@ func httpPostWithParamsHeaders(url string, val url.Values, headers map[string]st
 
 	defer resp.Body.Close()
 	result, err := ioutil.ReadAll(resp.Body)
-	if err != nil || string(result) == "" {
```

Comment:
I think you need to rebase and this change should not be visible in the release PR.

Reply:
That's weird, I cherry-picked your commit onto the release branch. Will do a rebase